src/main/java/StringCalculatorStep8.java

```diff
@@ -0,0 +1,17 @@
+import java.util.Arrays;
```

Comment:
Consider using List<String> instead of String[] to handle the input numbers more flexibly.